modules/services/cliphist.nix

```diff
@@ -46,5 +54,21 @@ in {
 
       Install = { WantedBy = [ cfg.systemdTarget ]; };
     };
+
+    systemd.user.services.cliphist-images = lib.mkIf cfg.allowImages {
+      Unit = {
+        Description = "Clipboard management daemon - images";
+        PartOf = [ "graphical-session.target" ];
+      };
+
+      Service = {
+        Type = "simple";
+        ExecStart =
+          "${pkgs.wl-clipboard}/bin/wl-paste --type image --watch ${cfg.package}/bin/cliphist store";
+        Restart = "on-failure";
+      };
+
+      Install = { WantedBy = [ cfg.systemdTarget ]; };
+    };
```

Comment:
This should just add the flags conditionally Instead of copying the entire unit.
e.g.:
```nix
"${pkgs.wl-clipboard}/bin/wl-paste}/bin/wl-paste ${lib.optionals cfg.allowImages "--type image"} ...
```

Reply:
without it, if I copy something from firefox, I just get the html entry in cliphist
```
nixos on  main [!] 
❯ cliphist list
905	<meta http-equiv="content-type" content="text/html; charset=utf-8"><img src="https://avatars.githubu
```